website/spec/Connections.md

```diff
@@ -379,6 +379,9 @@ selecting the `cursor` of each edge; Relay Modern began selecting
 `startCursor` and `endCursor` instead to save bandwidth (since it doesn't use any
 cursors in between).
 
+In the event that edges is an empty array, `PageInfo` must still return, with the
+`startCursor` and `endCursor` nulled repectivly.
+
```

Comment:
This is wrong? The cursors can do whatever it wants. Maybe should change this to say more like "it is suggested that in the event..."? @kassens